src/components/app.jsx

```diff
@@ -47,6 +47,12 @@ const App = React.createClass({
               </div>
             </section>
             {this.props.children}
+            <footer style={{marginTop: "5em"}}>
```

Comment:
Nothing related to React.

In this project there is no project-wide CSS stylesheet apart from bootstrap.css.

Why 😱? I use inline style when there is no need to generalize a style. If bootstrap 3 had spacing utilities I would have used them, but that's for bootstrap 4 ([spacing](https://v4-alpha.getbootstrap.com/utilities/spacing/))